docs/source/components.rst

```diff
@@ -0,0 +1,65 @@
+.. _chapter_components:
+
+===============
+Core Components
+===============
+
+The SDK has four core components but it is open to the contribution of any system that support the execution of scientific workflows on the Departiment of Energy high performance computing platforms.
+
+*  `Flux <http://flux-framework.org>`_. Workload management system (à la Slurm, PBS, LSF), with single-user and multi-user (a.k.a. system instance) modes.
+*  `Parsl <https://parsl-project.org>`_. Pure Python library for describing and managing parallel computation on one to many nodes.  It contains abstractions to support various parallel and distributed workloads: from bag-of-tasks to dataflow, short to long duration tasks, single core through to multi-node.
+*  `PSI/J <https://github.com/ExaWorks/psi-j-python>`_. The Portable Submission Interface for Jobs TODO: add description here.
```

Comment:
a Python library providing a single interface to a variety of HPC resource managers, including LSF, Flux, Slurm, Cobalt, and PBS.

Reply:
(That's a suggestion for eliminating the "TODO:")

Reply:
Link updated and added a short description.